brayns/common/PropertyMap.h

```diff
@@ -355,6 +363,9 @@ class PropertyMap
     const auto& getProperties() const { return _properties; }
     /** Merge this property map with properties from another. */
     void merge(const PropertyMap& input);
+    /** Take the values from another property map only for properties that
+        are known and compatible to this one. */
```

Comment:
if a property is incompatible it throws. Shall be mentioned for this and for `merge`

Reply:
That happens in both cases.